lib/rubygems/commands/query_command.rb

```diff
@@ -31,4 +35,13 @@ def initialize(name = 'query',
 
     add_query_options
   end
+
+  def description # :nodoc:
+    <<~DESCRIPTION
+      The query command is the basis for the list and search commands.
+
+      You should really use the list and search commands instead.  This command
+      is too hard to use.
+    DESCRIPTION
+  end
```

Comment:
Moving the description to the command itself seems like a good idea even if no longer necessary for the fix. Let's keep the previous `EOF` separator though. `DESCRIPTION` looks better but I prefer to update that consistently for all commands and to enforce good names when doing it with `rubocop`.